tests/backbone-computed-attributes.tests.js

```diff
@@ -78,6 +78,119 @@ suite("Backbone computed attributes", function() {
     });
   });
 
+  suite("Binding to collections", function() {
+    var TestModel;
+    setup(function() {
+      TestModel = Backbone.Model.extend({
+        initialize: function() {
+          this.createComputedAttribute({
+            attr: "Z",
+            get: function() {
+              return this.get("items").where({Y: "X"}).length;
+            },
+            bindings: [
+              { collection: this.get("items"), attribute: "Y" }
+            ]
+          });
+        }
+      });
+      _.extend(TestModel.prototype, ComputedAttributeMixin);
+    });
+
+    test("initialize and get computed attribute", function() {
+      var test = new TestModel({items: new Backbone.Collection()});
+      assert.equal(test.get("Z"), 0);
+    });
+
+    test("computed attribute works when bound to a collection", function() {
+      var test = new TestModel({items: new Backbone.Collection([new Backbone.Model({ Y: "X" }), new Backbone.Model({ Y: "?" })])});
+      assert.equal(test.get("Z"), 1);
+    });
+
+    test("bind to computed attribute on a collection works when item is added", function() {
+      var counter = 0;
+      var test = new TestModel({items: new Backbone.Collection()});
+      test.on("change:Z", function() { counter++; });
+      test.get("items").add(new Backbone.Model({ Y: "X" }));
+      assert.equal(counter, 1);
+      assert.equal(test.get("Z"), 1);
+    });
+
+    test("computed attribute does not fire multiple change events when dependencies are changed within an atomic block", function() {
+      var counter = 0;
+      var test = new TestModel({items: new Backbone.Collection()});
+      test.on("change:Z", function() { counter++; });
+      Backbone.atomic(function() {
+          test.get("items").add(new Backbone.Model({ Y: "X" }));
+          test.get("items").add(new Backbone.Model({ Y: "" }));
+          test.get("items").add(new Backbone.Model({ Y: "X" }));
+          test.get("items").add(new Backbone.Model({ Y: "" }));
+      });
+      assert.equal(test.get("Z"), 2);
+      assert.equal(counter, 1);
+    });
+
+    test("computed attribute does not fire change event when dependency is set to an unchanged value", function() {
+      var counter = 0;
+      var test = new TestModel({items: new Backbone.Collection()});
+      test.on("change:Z", function() { counter++; });
+      test.get("items").add(new Backbone.Model({ Y: "X" }));
+      assert.equal(counter, 1);
+      test.get("items").first().set("Y", "X");
+      assert.equal(counter, 1);
+    });
+
+    test("computed attribute does not fire change event when dependencies are changed but computed value remains the same", function() {
```

Comment:
when I first read this I thought the dependencies were the attributes, so for eg:
```
test.get("items").add(new Backbone.Model({ Y: "?" }));
test.get("items").first().set("Y", "");
```
Would this fire a change event?